package.json

```diff
@@ -54,6 +57,8 @@
     "lodash": "^4.17.19",
     "moment": "^2.24.0",
     "ndjson": "^1.5.0",
+    "node-sass": "^6.0.1",
+    "npm-run-all": "^4.1.5",
```

Comment:
These should be added as dev dependencies.

Reply:
yup - DERP